site/topics/genetic-operators/genetic-operators.rst

```diff
@@ -123,6 +123,55 @@ Bit Flip Mutation
 Genetic Operators for Integer Representations
 =============================================
 
+* Integer representations are those that consist of integer values
+
+
+Crossover
+---------
+
+* The crossovers used for binary representations are typically also used for integer representations
+
+
+Mutations
+---------
+
+Single Point Mutation
+^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Maybe worth a bullet point to mention something like swap mutations `(a,b) <- (b,a)`

Reply:
I was gonna bring it up in the permutation section, but the more the merrier